src/main/java/org/jenkinsci/plugins/gitclient/verifier/ManuallyProvidedKeyVerifier.java

```diff
@@ -34,53 +34,55 @@ public AbstractCliGitHostKeyVerifier forCliGit(TaskListener listener) {
             if (File.pathSeparatorChar
                     == ';') { // check whether on Windows or not without sending Functions over remoting
                 // no escaping for windows because created temp file can't contain spaces
```

Comment:
When a temporary file contains spaces on Windows, it creates problems for command line git, independent of any issues for the git client plugin.  A [warning is written to the build log](https://github.com/jenkinsci/git-client-plugin/blob/9ca1f0f85c80dd3c20626c1f3a516719f4ed496e/src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java#L240) if a Windows temporary directory is detected with spaces because of the known issue in some versions of command line git.